include/IMeshBuffer.h

```diff
@@ -134,11 +136,20 @@ GL_INVALID_OPERATION is generated by glVertexAttribPointer if size is GL_BGRA an
         {0xdeadbeefu,8,16,24,32}
     };
 
+	//! Checks whether given combination of attribute's type and component count is valid
+	/**@param type Type of an atrribute.
+	@param components Component count of an atrribute.
+	@returns true if such combination is valid or false otherwise.
+	*/
     inline bool validCombination(const E_COMPONENT_TYPE& type, const E_COMPONENTS_PER_ATTRIBUTE& components)
     {
         return vertexAttrSize[type][components]<0xdeadbeefu;
     }
 
+	//! Checks whether given attribute type is a normalized one.
```

Comment:
In OpenGL parlance Normalized-Integer means whether an integer value gets rescaled to [0,1] or [-1,1] float value respectively for unsigned and signed integer, by the time it reaches the shader input as a vertex attribute or texel from a texture.